File: Java/Produk.java
public class Produk { // Bikin class Produk
    // Atribut
    private String nama;
    private String merek;
    private int harga;
    private int stok;

    // Konstruktor default
    public Produk() {
        this.nama = "";
        this.merek = "";
        this.harga = 0;
        this.stok = 0;
    }

    // Konstruktor dengan parameter
    public Produk(String nama, String merek, int harga, int stok) {
        this.nama = nama;
        this.merek = merek;
        this.harga = harga;
        this.stok = stok;
    }

    // Setter dan Getter untuk setiap atribut

    public void setNama(String nama) {
        this.nama = nama;
    }

    public String getNama() {
        return this.nama;
    }

    public void setMerek(String merek) {
        this.merek = merek;
    }

    public String getMerek() {
        return this.merek;
    }

    public void setHarga(int harga) {
        this.harga = harga;
    }

    public int getHarga() {
        return this.harga;
    }

    public void setStok(int stok) {
        this.stok = stok;
    }

    public int getStok() {
        return this.stok;
    }

    // Method untuk print produk
    public void print() {
        System.out.println("Nama: " + this.nama);
        System.out.println("Merek: " + this.merek);
        System.out.println("Harga: " + this.harga);
        System.out.println("Stok: " + this.stok);
    }
}

File: C++/Produk.cpp
#include <iostream>
#include <string>
using namespace std;

class Produk { // Bikin class Produk
    private: // Atribut private
        string nama;
        string merek;
        int harga;
        int stok;

    public: // Method public
        Produk() { // Konstruktor default
        }

        Produk(string nama, string merek, int harga, int stok) { // Konstruktor dengan parameter
            // this-> untuk membedakan atribut dan parameter
            this->nama = nama;
            this->merek = merek; 
            this->harga = harga;
            this->stok = stok;
        }

        // Setter dan Getter untuk setiap atribut

        void setNama(string nama) {
            this->nama = nama;
        }
        string getNama() {
            return this->nama;
        }

        void setMerek(string merek) {
            this->merek = merek;
        }
        string getMerek() {
            return this->merek;
        }

        void setHarga(int harga) {
            this->harga = harga;
        }
        int getHarga() {
            return this->harga;
        }

        void setStok(int stok) {
            this->stok = stok;
        }
        int getStok() {
            return this->stok;
        }

        void print() { // Method untuk print produk
            cout << "Nama: " << this->nama << endl;
            cout << "Merek: " << this->merek << endl;
            cout << "Harga: " << this->harga << endl;
            cout << "Stok: " << this->stok << endl;
        }

        ~Produk() { // Destruktor
        }
};